src/tests/EdFi.DataManagementService.Tests.E2E/Features/References/UpdateInvalidReferenceValidation.feature

```diff
@@ -0,0 +1,226 @@
+Feature: Invalid Reference Validation
+    PUT requests validation for invalid references
+
```

Comment:
Once DMS-259 goes in, review the CreateReferenceValidation feature file and use the `Given the system has these descriptors` step to define all the descriptors you use, to prevent the tests breaking when we turn on descriptor validation.